migrations/000014_create_sorted_workloadtype.down.sql

```diff
@@ -0,0 +1,3 @@
+CREATE TYPE workloadtype AS ENUM ('daemonset', 'deployment', 'deploymentconfig', 'replicaset', 'replicationcontroller', 'statefulset');
```

Comment:
@upadhyeammit Do you think we preserve the order even though it was wrong for workloadtype enum?

Reply:
actually we have updated the first migration which creates the wrong order, so I thought to reflect on it

Reply:
I don't think the correct one ever got applied, because it was old?
I mean from a state pov, once we hit the down migration file